src/cls/rgw/cls_rgw.cc

```diff
@@ -4064,6 +4064,144 @@ static int rgw_get_bucket_resharding(cls_method_context_t hctx,
   return 0;
 }
 
+#define RGW_ATTR_COMPRESSION "user.rgw.compression" // from rgw_common.h
+#define RGW_ATTR_CRYPT_MODE "user.rgw.crypt.mode" // from rgw_common.h
+#define RGW_CRYPT_BLOCK_SIZE 4096 // from rgw_common.h
```

Comment:
please move this define into `cls/rgw/cls_rgw_types.h` (which is included by rgw_common.h) so we only have one definition. if someone needs to change it, that makes it clear that it affects both rgw and cls/rgw

Reply:
done